core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1458,6 +1456,16 @@ public void validateAuth(String workflowId, HttpHeaders headers) {
 		return decoded;
 	}
 
+
+	public Map<String, Object> validateAuthContext(HttpHeaders headers) {
+		if (!this.validateAuthContext ) {
+			return null;
+		}
+		String authString = headers.getRequestHeader("Authorization-Context").get(0);
```

Comment:
If the `Authorization-Context` header is empty, is `authString` here simply set to "" ?

Reply:
Authorization-Context is optional so if there is no string pass from previous system, we need to return empty string. @sangeetanadgir  could you confirm?

Reply:
@heajuenhwang @hueys  The above code snippet will be executed only when context is not null and not empty.So in case if its empty the authContext  map  variable  will be empty which we store in workflow instance. And we already have a condition check in GenericHttpTask class to pass the empty string to http call if authContext  of workflow is empty or null.